### DISCUSSION TITLE
My content disappears when i go back to the top

### DISCUSSION THREAD

**Discussion by Minozzzi** [idiotWu/smooth-scrollbar#525] [Q&A]
Hello everyone, I have a site in nextjs and I am initializing the smoothScrollbar in the application body, when I go to the bottom of the page and come back to the top my header disappears

https://www.loom.com/share/b75b2899758843489d5a4ade464535b4
What to do in this situation?

**Comment by sadeghbarati** [comment]
Hi, any online demo?

I can't access to loom content

**Comment by Minozzzi** [comment]
Yes!!
take a look: https://nh-site-natahouse.vercel.app/services

**Reply by sadeghbarati** [reply]
```html
<body>
  <div id="smooth-scrollbar" data-scrollbar>
    <main id="main-content">
      <!--- ALL YOUR CONTENT HERE --->
    </main>
  </div>
  <!-- position: fixed elements can go outside --->
</body>
```

can you follow this HTML structure to see if same happens again? (init smooth-scrollbar on another element other than body)

**Reply by Minozzzi** [reply]
The smooth scrolling was not applied

**Reply by Minozzzi** [reply]
![image](https://user-images.githubusercontent.com/61752597/229928378-7a7ac996-d988-4f84-82cd-e25c5394b646.png)

**Comment by Minozzzi** [comment]
I managed to solve it by placing a div under the body and initializing smooth scrolling in it, what was missing was the 100vh height for the error mentioned above
